services/web/server/src/simcore_service_webserver/meta_api_handlers_snapshots.py

```diff
@@ -312,37 +232,9 @@ async def _update_snapshot(
         # TODO: skip_return_updated
     )
 
-    data = SnapshotItem.from_snapshot(snapshot, url_for)
+    data = SnapshotItem.from_snapshot(snapshot, url_for, prefix=__name__)
     return enveloped_response(data)
 
 
-## projects/*/snapshots/*/parameters  -----------------------------
-
-
-@routes.get(
-    f"/{vtag}/projects/{{project_id}}/snapshots/{{snapshot_id}}/parameters",
-    name="get_snapshot_parameters_handler",
-)
-@login_required
-@permission_required("project.read")
-@handle_request_errors
-async def get_project_snapshot_parameters_handler(
-    request: web.Request,
-):
-    # pylint: disable=unused-variable
-    # pylint: disable=unused-argument
-    user_id, product_name = request[RQT_USERID_KEY], request[RQ_PRODUCT_KEY]
-
-    @validate_arguments
-    async def get_snapshot_parameters(
-        project_id: UUID,
-        snapshot_id: str,
-    ):
-        return {"x": 4, "y": "yes"}
-
-    params = await get_snapshot_parameters(
-        project_id=request.match_info["project_id"],  # type: ignore
-        snapshot_id=request.match_info["snapshot_id"],
-    )
-
-    return params
+rename_routes_as_handler_function(routes, prefix=__name__)
```

Comment:
are the names of the handles here importa? will renaming cause issues? If yes, also add a comment here as well.

Reply:
yes, the routes name are very important. Used to produce reverse urls or to test the API implementation vs the OAS